commands/meme.js

```diff
@@ -41,5 +41,35 @@ const Module = new Augur.Module()
 
             msg.channel.send({ files: [meme] });
         },
-    });
+    }).addCommand({
+        name: "Behold",
+        description: "creates a doofenshritz behold meme, with the arguments becoming your bottom text. Small chance of fire.",
+        permissions: (msg) => msg.channel.id == "121755900313731074" && msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
+        process: (msg) => {
+            function randomNumber(min, max) {
+                return Math.round(Math.random() * (max - min) + min);
+            }
+            //Roll a D20. on a 20 get a squirell. On a 1, elmo burns. anything else you get heinz
+            function backgroundImage(override) {
+                switch (override || randomNumber(1, 20)) {
+                    case 1: return `https://static1.srcdn.com/wordpress/wp-content/uploads/2020/05/Elmo-Flames-Meme.jpg?q=50&fit=crop&w=960&h=500`;
+                    case 20: return `https://i.imgflip.com/18kirh.jpg`;
+                    default: return `https://i.imgflip.com/zs91u.jpg`;
+                }
+            }
+            let { command, suffix } = u.parse(msg, true);
+            let content = `${Module.config.prefix}${command} ${backgroundImage()} Behold! ${suffix}`;
+
+            let fakeMsg = {
+              channel: msg.channel,
+              guild: msg.guild,
+              member: msg.member,
+              author: msg.author,
+              attachments: msg.attachments,
+              content: content,
+              cleanContent: content
+            };
+            Module.client.commands.execute("meme", fakeMsg, "");
```

Comment:
It shouldn't matter in this case, but best practice to pass the suffix to the `.execute()`.
```suggestion
            Module.client.commands.execute("meme", fakeMsg, content);
```